web/public/css/dashboard.less

```diff
@@ -62,11 +62,20 @@ label {
     padding: 46px 100px 0 100px;
 }
 
+
+.anchor-jump {
+  padding-top: 140px;
+  margin-top: -140px;
```

Comment:
Saw it, removed it just 10 min ago. I'll probably have to go js, which piss me a bit.